backend/text_extractor.py

```diff
@@ -0,0 +1,20 @@
+import glob
+
+import pytesseract
+from PIL import Image
+# class TextExtractor:
+#     def extract_text(img):
+pytesseract.pytesseract.tesseract_cmd = r"C:\Users\Jeremy\AppData\Local\Tesseract-OCR\tesseract.exe"
```

Comment:
Is it possible to get this value from a different file or source that isn't in source control?

Reply:
Yes, we can get this from a command line argument, a config file, etc.